README.md

```diff
@@ -4,34 +4,58 @@ Experimental support for VS Code Server in NixOS. The NodeJS by default supplied
 
 ## Installation
 
+### Flake
 ```nix
 {
-  imports = [
-    (fetchTarball "https://github.com/msteen/nixos-vscode-server/tarball/master")
-  ];
-
-  services.vscode-server.enable = true;
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixos-vscode-server.url ="github:iosmanthus/nixos-vscode-server/add-flake";
+  };
+
+  outputs = inputs@{self, nixpkgs, ...}: {
+    nixosConfigurations.some-host = nixpkgs.lib.nixosSystem rec {
+        system = "x86_64-linux";
+        # For more information of this field, check:
+        # https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/eval-config.nix
+        modules = [
+          ./configuration.nix
+          {
+            imports = [ inputs.auto-fix-vscode-server.nixosModules.system ];
+            services.auto-fix-vscode-server.enable = true;
+          }
+        ];
+      };
+  };
 }
 ```
 
-And then enable them for the relevant users:
-
-```
-systemctl --user enable auto-fix-vscode-server.service
-```
-
 ### Home Manager
 
 ```nix
 {
-  imports = [
-    "${fetchTarball "https://github.com/msteen/nixos-vscode-server/tarball/master"}/modules/vscode-server/home.nix"
-  ];
-
-  services.vscode-server.enable = true;
+    inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixos-vscode-server.url ="github:iosmanthus/nixos-vscode-server/add-flake";
+  };
+
+  outputs = inputs@{self, nixpkgs, ...}: {
+    nixosConfigurations.some-host = nixpkgs.lib.nixosSystem rec {
+        system = "x86_64-linux";
+        # For more information of this field, check:
+        # https://github.com/NixOS/nixpkgs/blob/master/nixos/lib/eval-config.nix
+        modules = [
+          ./configuration.nix
+          {
+            home-manager = {
+              user.iosmanthus = {
+                imports = [ 
+                  inputs.nixos-vscode-server.nixosModules.homeManager;
+                ];
+              };
+            };
+          }
+        ];
+      };
+    };
 }
-```
-
-## Usage
-
-When the service is enabled and running it should simply work, there is nothing for you to do.
+```
```

Comment:
This is likely just me being ignorant, but I had troubles getting this to work without following these steps.

````suggestion
```

## Usage

This service only works if the VSCode server has yet to be installed. If the
server has already been installed, remove `~/.vscode-server`, restart the
service, and then try to connect to the VSCode server from another client again.
The service can be restarted with
`systemctl --user restart auto-fix-vscode-server` or by rebooting the machine.
````